Source/OnlineSubsystemEpic/Private/OnlineSubsystemEpicModule.cpp

```diff
@@ -92,16 +92,16 @@ void FOnlineSubsystemEpicModule::StartupModule()
 	FString LibraryPath;
 #if PLATFORM_WINDOWS
 #if PLATFORM_32BITS
-	LibraryPath = FPaths::Combine(*BaseDir, TEXT("Source/ThirdParty/OnlineSubsystemEpicLibrary/x86/EOSSDK-Win32-Shipping.dll"));
+	LibraryPath = FPaths::Combine(*BaseDir, TEXT("Source/ThirdParty/OnlineSubsystemEpicLibrary/Bin/EOSSDK-Win32-Shipping.dll"));
```

Comment:
LMK if these paths are correct, they should probably mirror the paths that are provided by the EOS SDK

Reply:
They are indeed correct. I changed them in the ThirdPartyPlugin, but forgot to change them when loading.